app/views/entities/new.html.erb

```diff
@@ -11,7 +11,12 @@
     <%= form.label :amount %>
     <%= form.text_field :amount, class: 'form-control', placeholder: 12, type: "number", step: ".01" %>
   </div>
-  
+
+    <div class="mb-3">
+    <%= form.label :Category %>
+    <%= form.text_field :name, class: 'form-control', value: @group.name %>
+  </div>
+
```

Comment:
[OPTIONAL] Last but not least. Kindly put your own name here. Instead of nickname.

<img width="633" alt="Screen Shot 2022-10-21 at 19 55 43" src="https://user-images.githubusercontent.com/69011963/197226986-78a3f7b8-2278-4431-8b52-d6e903a30295.png">